public/components/custom_panels/home.tsx

```diff
@@ -147,7 +147,7 @@ export const Home = ({
     return http.get(`${CUSTOM_PANELS_API_PREFIX}/panels/${id}`);
     // .then((res) => res.operationalPanel)
     // .catch((err) => {
-    //   console.error('Issue in fetching the operational panel to duplicate', err);
+    //   console.error('Issue in fetching the Observability Dashboard to duplicate', err);
     // });
```

Comment:
Can be removed, if not being used.

Reply:
Want to keep these until Toast re-imiplementation.  THat's where I'll trap thte error handlers correctly.